tests/mixin.flowchart.tests.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable max-classes-per-file */
+import { expect } from "chai";
+import { mix } from "mixwith";
+
+import { FlowchartEntityMixin, FlowchartItemMixin, InMemoryEntity } from "../src/entity";
+
+class FlowchartEntity extends mix(InMemoryEntity).with(FlowchartEntityMixin) {}
+
+class FlowchartItem extends mix(InMemoryEntity).with(FlowchartItemMixin) {}
+
+describe("Flowchart Mixins", () => {
+    it("flowchart item can be added", () => {
+        const entity = new FlowchartEntity();
+        entity.addUnit(new FlowchartItem());
+        expect(entity.units.length).to.be.equal(1);
+    });
+
+    it("flowchart item can be added inbetween", () => {
```

Comment:
Re https://www.grammarly.com/blog/in-between-or-inbetween/, maybe just use "added between two other units"